Vaadin 19 beta 1 project using npm 7 does not compile

### Description of the bug / feature
If you use npm with a Vaadin 19 beta1 project, `npm install` fails with
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: no-name@undefined
npm ERR! Found: webpack@4.42.0
npm ERR! node_modules/webpack
npm ERR!   dev webpack@"4.42.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer webpack@">=4.44.2" from webpack-manifest-plugin@3.0.0
npm ERR! node_modules/webpack-manifest-plugin
npm ERR!   dev webpack-manifest-plugin@"3.0.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/travis/.npm/eresolve-report.txt for a full report.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2021-02-10T11_59_17_503Z-debug.log
```

### Minimal reproducible example
Download a Vaadin 19 project using npm from start.vaadin.com and run `mvn`

### Expected behavior
It works

### Versions:
    - Vaadin / Flow version: 19 beta 1

--- a/flow-server/src/main/java/com/vaadin/flow/server/frontend/NodeUpdater.java
+++ b/flow-server/src/main/java/com/vaadin/flow/server/frontend/NodeUpdater.java
@@ -295,7 +295,7 @@ public abstract class NodeUpdater implements FallibleCommand {
         defaults.put("typescript", "4.0.3");
         defaults.put("ts-loader", "8.0.12");
 
-        defaults.put("webpack", "4.42.0");
+        defaults.put("webpack", "4.46.0");
         defaults.put("webpack-cli", "3.3.11");
         defaults.put("webpack-dev-server", "3.11.0");
         defaults.put("compression-webpack-plugin", "4.0.1");

--- a/flow-server/src/test/java/com/vaadin/flow/server/frontend/NodeUpdaterTest.java
+++ b/flow-server/src/test/java/com/vaadin/flow/server/frontend/NodeUpdaterTest.java
@@ -173,7 +173,7 @@ public class NodeUpdaterTest {
                 "3.3.10");
         nodeUpdater.updateDefaultDependencies(packageJson);
 
-        Assert.assertEquals("4.42.0", packageJson
+        Assert.assertEquals("4.46.0", packageJson
                 .getObject(NodeUpdater.DEV_DEPENDENCIES).getString("webpack"));
     }
 

